common/app/routes/challenges/redux/reducer.js

```diff
@@ -81,7 +82,13 @@ const mainReducer = handleActions(
     }),
     [types.updateTests]: (state, { payload: tests }) => ({
       ...state,
-      tests
+      tests,
+      isChallengePassed: (tests.length > 0 &&
```

Comment:
I see what you mean. 

Instead of isChallengePassed it should be isChallengeModalOpen.